.travis.yml

```diff
@@ -71,6 +71,7 @@ before_install:
     - export TRAVIS_INSTALL=$HOME/travis/install
     ## Disable security protection so CMA will work
     - sudo sysctl -w kernel.yama.ptrace_scope=0
+    - ulimit -n 2048
```

Comment:
Our users are going to run into the same limitation.  Might be better to reduce the number of contexts created in the unit tests instead.